pkg/omf/omf.fish

```diff
@@ -55,9 +55,9 @@ function omf -d "Oh My Fish"
     case "c" "cd"
       switch (count $argv)
         case 1
-          omf.cd
+          omf.packages.cd
```

Comment:
That's not the same thing, I want to go to `omf root folder`, not to the plugin named `omf`. :)